test/functional/mining_basic.py

```diff
@@ -237,11 +237,19 @@ def assert_submitblock(block, result_str_1, result_str_2=None):
         bad_block = copy.deepcopy(block)
         bad_block.vtx[0].vin[0].prevout.hash += 1
         bad_block.vtx[0].rehash()
+        self.log.info("getblocktemplate: Test block with missing coinbase")
         assert_template(node, bad_block, 'bad-cb-missing')
 
+        self.log.info("submitblock: Test empty block")
```

Comment:
6993836cb6488afa48922f19778ecef6d122bb69 nit: adding these log messages could be its own commit to de-obfuscate what it's actually testing.